cmd/commands/compose.go

```diff
@@ -7,6 +7,8 @@ import (
 	"os"
 	"strings"
 
+	"github.com/docker/ecs-plugin/pkg/amazon/cloudformation"
+
```

Comment:
Should be grouped with the imports below, or the other `ecs-plugin` import would be grouped with this one